<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>725</width>
    <height>618</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Practice3</string>
  </property>
  <widget class="QPushButton" name="exitBtn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>560</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter an Expression</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>10</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Syntax Tree</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Derivation Process</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>350</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>AST Tree</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="derivationTxt">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>241</width>
     <height>391</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;MS UI Gothic&quot;;
font: 700 10pt &quot;Monospac821 BT&quot;;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>241</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="okBtn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clearBtn">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>560</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="syntaxTree">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>30</y>
     <width>401</width>
     <height>311</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;MS UI Gothic&quot;;
font: 700 10pt &quot;Monospac821 BT&quot;;</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="astTree">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>370</y>
     <width>401</width>
     <height>221</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;MS UI Gothic&quot;;
font: 700 10pt &quot;Monospac821 BT&quot;;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>exitBtn</sender>
   <signal>clicked()</signal>
   <receiver>Widget</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>70</x>
     <y>575</y>
    </hint>
    <hint type="destinationlabel">
     <x>127</x>
     <y>596</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearBtn</sender>
   <signal>clicked()</signal>
   <receiver>derivationTxt</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>178</x>
     <y>568</y>
    </hint>
    <hint type="destinationlabel">
     <x>182</x>
     <y>520</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearBtn</sender>
   <signal>clicked()</signal>
   <receiver>lineEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>180</x>
     <y>576</y>
    </hint>
    <hint type="destinationlabel">
     <x>180</x>
     <y>54</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearBtn</sender>
   <signal>clicked()</signal>
   <receiver>astTree</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>185</x>
     <y>570</y>
    </hint>
    <hint type="destinationlabel">
     <x>349</x>
     <y>550</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearBtn</sender>
   <signal>clicked()</signal>
   <receiver>syntaxTree</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>174</x>
     <y>572</y>
    </hint>
    <hint type="destinationlabel">
     <x>418</x>
     <y>286</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
